### ISSUE TITLE
Failure on searching epub with pandoc deprecation warning

### ISSUE THREAD

**Issue by eidoom** [phiresky/ripgrep-all#115]
Hello,
I have encountered the following failure
```shell
$ rga "<search>" <filename>.epub
<filename>.epub: preprocessor command failed: '"/home/<user>/.local/bin/rga-preproc" "<filename>.epub"':
-------------------------------------------------------------------------------
adapter: pandoc
[WARNING] Deprecated: --atx-headers. Use --markdown-headings=atx instead.
parseSpine
Error: subprocess failed: ExitStatus(ExitStatus(16384))
-------------------------------------------------------------------------------
```
I am using pandoc 2.11.2. My OS is Fedora 34.

**Comment by morganmay** [comment]
I'm experiencing the same preprocessor issue with EPUB files on Arch Linux with `ripgrep-all 0.9.6-3` and `pandoc 2.17.1.1-15` 

```
-------------------------------------------------------------------------------
adapter: pandoc
[WARNING] Deprecated: --atx-headers. Use --markdown-headings=atx instead.
Couldn't extract ePub file
Error: subprocess failed: ExitStatus(unix_wait_status(16384))
-------------------------------------------------------------------------------
```

**Comment by gbstan** [comment]
Any chance a new version of ripgrep-all could be released that fixes this issue? I don't know how to compile from source and ripgrep-all is one of the tools I use regularly.

**Comment by mar-schl** [comment]
install pandoc version 2.19.2

**Comment by phiresky** [comment]
i'll probably release the beta of 1.0 soon

**Comment by lafrenierejm** [comment]
> Any chance a new version of ripgrep-all could be released that fixes this issue? I don't know how to compile from source and ripgrep-all is one of the tools I use regularly.

[v1.0.0-alpha.5](https://github.com/phiresky/ripgrep-all/releases/tag/v1.0.0-alpha.5) is the latest pre-release version as of this writing.

**Comment by phiresky** [comment]
Fixed in 0.10
